packages/synapse-react-client/src/components/Evaluation/CreatedOnByUserDiv.tsx

```diff
@@ -1,36 +1,18 @@
 import React from 'react'
-import { useSynapseContext } from '../../utils/context/SynapseContext'
 import { UserBadge } from '../UserCard/UserBadge'
+import { formatDate } from '../../utils/functions/DateFormatter'
+import dayjs from 'dayjs'
 
 export type CreatedOnByUserDivProps = {
   userId: string
   date: Date
 }
 
-const dateFormatOptionLocal: Intl.DateTimeFormatOptions = {
-  timeZoneName: 'short',
-}
-const dateFormatOptionUTC: Intl.DateTimeFormatOptions = {
-  timeZone: 'UTC',
-  timeZoneName: 'short',
-}
-
-export const CreatedOnByUserDiv: React.FunctionComponent<
-  CreatedOnByUserDivProps
-> = ({ userId, date }) => {
-  const { utcTime } = useSynapseContext()
+export function CreatedOnByUserDiv(props: CreatedOnByUserDivProps) {
+  const { userId, date } = props
   return (
     <div className="created-on">
-      <span>
-        Created on{' '}
-        {date
-          .toLocaleDateString(
-            undefined,
-            utcTime ? dateFormatOptionUTC : dateFormatOptionLocal,
-          )
-          .replace(',', '')}{' '}
-        by{' '}
-      </span>
```

Comment:
This ended up showing text like "Created on 3/24/2022 UTC"--use the utility to fix this